lib/cmd/fhc/login.js

```diff
@@ -167,4 +168,14 @@ function saveLogin(username, cookie, domain, targetUrl, cb) {
       return cb(err);
     });
   });
-}
\ No newline at end of file
+}
+
+function configureProxy() {
+  var proxy = fhc.config.get("proxy");
+  
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)